packaging/format-docs.py

```diff
@@ -81,14 +93,21 @@ def format_docs(version, collectionName, types):
                         elif 'Require' in attributes:
                             defaultValue = '*(required)*'
 
-                        print(f'| {prop["PropertyName"]} | {defaultValue} | {prop["UserFriendlyType"]} | {prop["Description"]} |')
+                        print(f'| {prop["PropertyName"]} | {defaultValue} | {typeName} | {prop["Description"]} |')
                     else:
-                        print(f'| {prop["PropertyName"]} | {prop["DefaultValue"] or ""} | {prop["UserFriendlyType"]} | {prop["Description"]} |')
+                        print(f'| {prop["PropertyName"]} | {prop["DefaultValue"] or ""} | {typeName} | {prop["Description"]} |')
+
+    if len(relatedEnums) > 0:
+        print('\n# Related enums:\n')
+        for relatedEnum in relatedEnums:
+            values = [value['Value'] for value in relatedEnum["Values"]]
+            print(f"#### {relatedEnum['Name']}")
+            print(f"{', '.join(values)}\n")
```

Comment:
```suggestion
            print(f"Possible values: {', '.join(values)}\n")
```

Something this this should be added, can't leave the values bare